Makefile.toml

```
@@ -484,7 +484,7 @@ done'''
 ]
 
 [tasks.build-tools]
-dependencies = ["fetch"]
+dependencies = ["setup", "fetch-sources"]
```

just checking, is this something to cache as well?
```suggestion
dependencies = ["setup", "fetch-sources", "fetch-vendored"]
```

---

I was thinking about caching `.gomodcache` but it really only provides about a 2-3 second benefit right now with caching it and would require us to have a separate cache step that is scoped to just the host-ctr and ecs-init `go.sum` files.